changelog/unreleased/pull-272

```diff
@@ -0,0 +1,12 @@
+Enhancement: can now listen on a unix socket
+
+If `--listen unix:/tmp/foo` is passed, the server will listen on a unix socket. This is triggered by the prefix `unix:`.
```

Comment:
To let rest-server listen on a unix socket, prefix the socket filename with `unix:` and pass it to the `--listen` option, for example `--listen unix:/tmp/foo`.

Reply:
Modified as suggested.